util/ss58.test.ts

```diff
@@ -32,27 +32,36 @@ for (
 }
 
 Deno.test("ss58.encode invalid public key length", () => {
-  assertInstanceOf(
-    ss58.encode(0, alice.publicKey.slice(0, 30)),
-    ss58.InvalidPublicKeyLengthError,
+  assertThrows(
+    () => ss58.encode(0, alice.publicKey.slice(0, 30)),
+    ss58.InvalidPublicKeyLengthError.prototype.message,
```

Comment:
`assertThrows` can take an error subclass:```suggestion
    ss58.InvalidPublicKeyLengthError,
```

Reply:
OH AWESOME. Never saw that it had that overload. Nice!